extensions/interactions/GraphInput/directives/GraphInput.js

```diff
@@ -256,9 +275,33 @@ oppia.directive('graphViz', [
           $scope.VERTEX_RADIUS = graphDetailService.VERTEX_RADIUS;
           $scope.EDGE_WIDTH = graphDetailService.EDGE_WIDTH;
 
+          if ($scope.canEditGraph === undefined ||
```

Comment:
Why not just pass $scope.interactionIsActive directly then?

Reply:
I tried passing that but when the graph interaction is created, its value is not defined and during this case its value should actually be true and assigning it as true gave errors, so I put a third variable

Reply:
That sounds ... hacky. Why is it not defined when the graph interaction is created? It sounds like that's the thing that needs to be looked into and fixed.

Reply:
Ok, will do that.